### PR TITLE
Develop Latvian translations

### PR DESCRIPTION
**Pull request by gatisr**
## Fixes Or Enhances

**Make sure that you've checked the boxes below before you submit PR:**
- [x] Tests exist or have been written that cover this particular change.

@go-playground/validator-maintainers

### CODE REVIEW

File: translations/lv/lv.go [hunk 1]
```diff
@@ -0,0 +1,1399 @@
...
+			override:    false,
+		},
+		{
+			tag:         "alphanum",
+			translation: "{0} jāsatur tikai simboli no alfabēta vai cipari (Alphanumeric)",
+			override:    false,
+		},
+		{
+			tag:         "numeric",
+			translation: "{0} jāsatur derīga numeric vērtība",
```
Comment (zemzale) [inline translations/lv/lv.go:976]: This probably should be `jāsatur tikai cipari`

File: translations/lv/lv.go [hunk 2]
```diff
@@ -0,0 +1,1399 @@
...
+			override:    false,
+		},
+		{
+			tag:         "number",
+			translation: "{0} jāsatur derīgs numurs",
```
Comment (zemzale) [inline translations/lv/lv.go:981]: This probably should be `jāsatur dreīgs skaitlis`

File: translations/lv/lv.go [hunk 3]
```diff
@@ -0,0 +1,1399 @@
...
+					log.Printf("warning: error translating FieldError: %#v", fe)
+					return fe.(error).Error()
+				}
+
+				return t
+			},
+		},
+		{
+			tag:         "postcode_iso3166_alpha2",
+			translation: "{0} does not match postcode format of {1} country",
```
Comment (zemzale) [inline translations/lv/lv.go:1328]: Missed translation. Suggesting something like 
`neatbilst {1} valsts pasta indeksa formātam`

File: translations/lv/lv.go [hunk 4]
```diff
@@ -0,0 +1,1399 @@
...
+					log.Printf("warning: error translating FieldError: %#v", fe)
+					return fe.(error).Error()
+				}
+
+				return t
+			},
+		},
+		{
+			tag:         "postcode_iso3166_alpha2_field",
+			translation: "{0} does not match postcode format of country in {1} field",
```
Comment (zemzale) [inline translations/lv/lv.go:1342]: Same as the other post-code translation.

**Review by zemzale — CHANGES_REQUESTED** [review]
Looks good. Just a few changes that should be made.

**Review by zemzale — APPROVED** [review]
Looks good now. @deankarn This can be merged if there are no problems with the implementation, but the translations look good.

**Review by deankarn — APPROVED** [review]

**Pull request merged**